README.md

```diff
@@ -1,6 +1,8 @@
 # Chart Testing
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Go Report Card](https://goreportcard.com/badge/github.com/helm/chart-testing)](https://goreportcard.com/report/github.com/helm/chart-testing)
+[![CircleCI](https://circleci.com/gh/helm/chart-testing/tree/master.svg?style=svg)](https://circleci.com/gh/helm/chart-testing/tree/master)
```

Comment:
This badge isn't loading for me at the moment, but it seems like it's just that Circle is down.

Reply:
Indeed, CircleCI is down.